lib/WP_Auth0_DBManager.php

```diff
@@ -425,7 +425,7 @@ public function get_auth0_users( $user_ids = null ) {
 		$results = get_users( $query );
 
 		if ( $results instanceof WP_Error ) {
-			WP_Auth0_ErrorManager::insert_auth0_error( 'findAuth0User', $userRow );
+			WP_Auth0_ErrorManager::insert_auth0_error( __METHOD__, $results->get_error_message() );
```

Comment:
Presume ` $results->get_error_message() ` is more useful than `$userRow` ?

Reply:
`$userRow` was never defined here